pos_config_show_accounting/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This module allows to show the accounting settings in the POS Configuration
+when the accounting module is installed. As of Odoo 11, it would only be
+visible when the Account Accountant module is available (which is available
+in EE only).
```

Comment:
Do this fit into OCA then?

Reply:
See https://github.com/odoo/odoo/blob/11.0/addons/point_of_sale/models/pos_config.py#L64

Reply:
> Do this fit into OCA then?

This module is showing feature that are hidden, if user is using CE, and not EE. so yes !
Maybe, we could update DESCRIPTION.rst file to be more clear ?

Reply:
Oh. I understand now :sweat_smile: